src/Domain/Model/Skill.php

```diff
@@ -48,4 +48,23 @@ public function removeResource($resourceId)
             }
         }
     }
+
+    public function addResourceVote($resourceId, ResourceVote $resourceVote) {
```

Comment:
Also avoid comments ... if you need to use comments it means that the code is not self-explanatory - which it is now ;)